pkg/tinydtls/include/sock_dtls_types.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @brief   tinydtls-specific types and functions definitions
+ *
+ * @author  Aiman Ismail <muhammadaimanbin.ismail@haw-hamburg.de>
+ * @author  Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#ifndef SOCK_DTLS_TYPES_H
+#define SOCK_DTLS_TYPES_H
+
+#include "dtls.h"
+#include "net/sock/udp.h"
+#include "net/credman.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef SOCK_DTLS_MBOX_SIZE
+#define SOCK_DTLS_MBOX_SIZE     (8)         /**< Size of mailbox */
+#endif
+
+/**
+ * @brief Information about DTLS sock
+ */
+struct sock_dtls {
+    dtls_context_t *dtls_ctx;               /**< sock context */
+    sock_udp_t *udp_sock;                   /**< Underlying UDP sock to use */
+    mbox_t mbox;                            /**< Mailbox for messaging */
+    msg_t mbox_queue[SOCK_DTLS_MBOX_SIZE];  /**< Queue of mailbox */
```

Comment:
`Queue for struct sock_dtls::mbox` (not sure this is the right notation with non-typedef'd structs, please check the output of `make doc`)

Reply:
Yup, that's correct. Doxygen links correctly to `sock_dtls::mbox`.